frontends/web/src/components/TaskOwnerPageComponents/Datasets.js

```diff
@@ -265,9 +265,10 @@ const Datasets = (props) => {
                               >
                                 {[
                                   "None",
-                                  ...[
-                                    ...Array(props.task.cur_round).keys(),
-                                  ].map((i) => i + 1),
+                                  ...Array.from(
+                                    { length: props.task.cur_round },
+                                    (x, i) => i + 1
+                                  ),
```

Comment:
thought the nested array spreading might've looked a bit ugly, so changed it to use `Array.from`